samza-core/src/main/scala/org/apache/samza/storage/ContainerStorageManager.java

```diff
@@ -329,18 +345,14 @@ public ContainerStorageManager(
   /**
    *  Creates SystemConsumer objects for store restoration, creating one consumer per system.
    */
-  private static Map<String, SystemConsumer> createConsumers(Map<String, Set<SystemStream>> systemStreams,
+  private static Map<String, SystemConsumer> createConsumers(Set<String> systems,
```

Comment:
can we give a meaningful name instead of systems? I see the call site passes `containerSideInputSystems` and `containerChangelogSystems`. so maybe keep `storeSystems` or anything that conveys more information than systems :)